packages/wallet/backend/knexfile.js

```diff
@@ -20,6 +20,20 @@ module.exports = {
     }
   },
 
+  rafiki_backend: {
```

Comment:
why is this added?

Reply:
I added this so that I can migrate assets to scale 9 in the `assets` table which is in `rafiki_backend` db.

Reply:
add an env variable instead with the connection string